### ISSUE TITLE
train question

### ISSUE THREAD

**Issue by sulei1998** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#294]
Hi! Thanks for your great work! Recently, I am trying to use your network to train a detector for detecting lower limb joints (including feet). I have conducted a complete training on the cocowhlobody dataset. The effect looks good on some images, but it doesn't seem to work on some images. I wonder if there is a problem with the resolution of my network input? My training uses the default 384x384 resolution. Is there any suggestion to improve my detection accuracy?

**Good example**
![image](https://user-images.githubusercontent.com/56378342/209669590-3ae7b0ae-5394-4b32-b50c-7e0ae026d2fb.png)

**Bad example**
![image](https://user-images.githubusercontent.com/56378342/209669640-e58dcea4-972d-42ae-8999-e3373bd1c62b.png)

**Comment by Daniil-Osokin** [comment]
Hi! I believe it is not due to the resolution, because person in "bad example" is big enough. You can try to visualize keypoints and pafs heatmaps (#235) to get some information what went wrong: either keypoints were not detected or pafs, or they were failed to be grouped into the instance due to occlusion of root keypoint (I guess it is in the center between hips in this case).

**Comment by sulei1998** [comment]
> Hi! I believe it is not due to the resolution, because person in "bad example" is big enough. You can try to visualize keypoints and pafs heatmaps (#235) to get some information what went wrong: either keypoints were not detected or pafs, or they were failed to be grouped into the instance due to occlusion of root keypoint (I guess it is in the center between hips in this case).

Hi! I visualize the heatmaps, it seems that the network do not detect the root keypoints. As you guess, in my setting, the root keypoints is in the center between hips.

![image](https://user-images.githubusercontent.com/56378342/209792635-4d303329-fd5d-464a-b383-89b090f6fc13.png)

In my test, the detection effect of this network on such pictures is not very good. Can you give me some advice?
![image](https://user-images.githubusercontent.com/56378342/209794518-d3a52589-1818-4ece-81b7-b06c1bdb1dcf.png)


In addition, I also trained using Human3.6M data, and the test still uses the validation set of the cocowholebody. Unlike training using the cocowholebody training set, it seems that the AP trained on this dataset has always been 0. When i train on cocowholebody, the AP can reach 30%+. Is this because the Human3.6M data is relatively simple?

**Comment by Daniil-Osokin** [comment]
Think so. If you have trained on "images from the lab" environment only, network often overfits. You may try to change background somehow, when training only on Human3.6M dataset.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.
